lib/CL/pocl_llvm_spirv.cc

```diff
@@ -104,6 +104,20 @@ static void handleInOutPathArgs(bool &keepPath, char *Path, char *HiddenPath,
 extern "C" int pocl_reload_program_bc(char *program_bc_path, cl_program program,
                                       cl_uint device_i);
 
+static bool getMaxSpirvVersion(pocl_version_t &MaxVersion,
+                               const char *DevVersions) {
+  MaxVersion.major = 1;
+  MaxVersion.minor = 0;
+  if (DevVersions) {
+    std::string SpirvVersions{DevVersions};
+    if (SpirvVersions.find("SPIR-V_1.") == 0) {
```

Comment:
AFAIK, It's not established that highest version appears first. Maybe setup this as invariant in comment in `_cl_device_id::supported_spir_v_versions`'s declaration.

Reply:
This is to @franz

Reply:
OK, fixed. Please drop my commits from this PR (changes to them were necessary).